server/utils/jwt.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require("jsonwebtoken");
+const secret = process.env.ACCESS_TOKEN_SECRET;
+
+let createToken = (payload) => {
+  // payload should be object
+  return jwt.sign(payload, secret, { expiresIn: "1m" });
```

Comment:
Since the token's being used to keep the user logged in, one minute might be a bit short of a timeout to use.

Reply:
Actually, I want to add a refresh token. After I finished the socket, I will come back to improve this part. One minute for sure is too short.